tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -1027,8 +1043,8 @@ Then('the expiration date shown on the webUI should be {string} days',
       .sharingDialog()
       .getExpirationDateFromInputField()
     assert.strictEqual(
-      expectedDateString,
       dateStringFromInputField,
+      expectedDateString,
```

Comment:
redundant change!

Reply:
i dont think so. the assertion had the expected and the actual strings interchanged. As I am working in the same feature, i just corrected that